app/src/main/res/values/strings.xml

```diff
@@ -67,6 +67,11 @@
     <string name="feedback_save_image_success">Photo saved successfully.</string>
     <string name="feedback_save_image_error">Something was wrong. Please try again.</string>
 
+    <string name="feedback_benchmark_button">Benchmark</string>
+    <string name="feedback_benchmark_try_again">Try again</string>
+    <string name="feedback_benchmark_error">Uh-oh, an error occurred!</string>
+    <string name="feedback_benchmark_empty">No classifications found.</string>
```

Comment:
No results found. ?

Think it fits better with the login call to action: "Send feedback and compare the results with other services."